lapis/storage.py

```diff
@@ -0,0 +1,81 @@
+from usim import time
+
+from typing import Optional
+
+from lapis.utilities.cache_algorithm_implementations import cache_algorithm
+from lapis.utilities.cache_cleanup_implementations import cache_cleanup
+
+
+class Storage(object):
+
+    __slots__ = ("name", "sitename", "storagesize", "usedstorage", "content")
+
+    def __init__(
+        self, name: str, sitename: str, storagesize: int, content: Optional[dict] = None
```

Comment:
# storagesize

What is the unit being used for `storagesize`? You might consider using `float` instead of `int`.
Further please check the `Capacity` classes from usim.

# content

Is `content` the right name here? If we are talking about storage, it might be better to talk about files?

Reply:
Hm... we should make a decision on this. If I see correctly, we currently use ``GB``. I opened a ticket to standardise the units: #58.